swagger.yaml

```diff
@@ -1023,7 +1023,7 @@ paths:
         201:
           description: Success.
           schema:
-            $ref: '#/definitions/host'
+            $ref: '#/definitions/host_registration_response'
```

Comment:
I think this name change is not backwards compatible, due to the python/go clients

Reply:
It is not the name, it is the structure. `host_registration_response` is `host` with the addition of the new structure. That is, a `host` fits into a `host_registration_response`. And because it is at the root of the JSON (as opposed to a named property), it will be parsed correctly.